### PR TITLE
Add draft Steam page for Emergence game concept

### PR DESCRIPTION
**Pull request by alice-i-cecile**
This PR presents the first draft text for a Steam page for Emergence. I find this format very helpful: it helps align the team around a shared vision while ensuring that this is clearly communicated and compelling to players. Plus, you can actually use it when it comes time to prepare for launch! 

I've set up [mdbook](https://rust-lang.github.io/mdBook/index.html) to use for our game design notes: the version controlled nature of them makes them much easier to track and collaborate on than scattered discussions or Google Docs.

### CODE REVIEW

File: design/src/steam-page.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+But when you're only a few centimeters in length, the world is a terribly unstable place!
+A rain storm becomes a great flood, predators are terrifying monsters, a fallen tree can level an entire district, and a dead animal presents an unprecedented windfall of nutrients.
+Day and night become your metronome, while the changing seasons force you to completely rethink your entire economy.
+Unchecked, disease can ravage your populace, while your very resource stores are ripe to be raided!
+**Robustness** is key.
+If you want to gather the raw materials needed to progress, much less combine them into the end products needed to take your colony to the next level via complex supply chains, you must learn to take advantage of your good fortune, while ensuring you can survive and adapt to the misfortune.
+
+Of course, not all problems are mere bad luck.
+As your predecssors learned, the most insidious probles are often those of your own making.
```
Comment (bjorn3) [inline design/src/steam-page.md:42]: *problems

**Review by TimJentzsch — COMMENTED** [review]
Overall I like it, but I would suggest to make it a bit easier to digest:

- I would recommend to avoid long and complex sentences and break them down into multiple smaller ones. For example this sentence:

  > If you want to gather the raw materials needed to progress, much less combine them into the end products needed to take your colony to the next level via complex supply chains, you must learn to take advantage of your good fortune, while ensuring you can survive and adapt to the misfortune.

- I think it's best to reduce the number of special terms to a minimum and remove those that are not required to understand the gist of the game.

- I would divide the long description into several small sections, each with a heading and an image (of course you can add the images later). People who are just skipping through the Steam discovery queue will probably not read a long text, but they can still skim over the section headings and look at the images to get a feel for the game and see if they might like it. As an example, [here's a random game on Steam that uses a similar format for their description](https://store.steampowered.com/app/518790/theHunter_Call_of_the_Wild/).

**Comment by alice-i-cecile** [comment]
Good feedback!

I'll prepare the next draft to try and tackle that.

**Review by bzm3r — COMMENTED** [review]
I like it as a first draft! Do agree that this is helpful in setting long term goals. 

I think @TimJentzsch raises some really good points though, so their comments are my comments 😆

**Comment by alice-i-cecile** [comment]
Merging as a first draft: feel free to make future PRs to tweak this!

**Pull request merged**